extract/clien/main.py

```
@@ -220,16 +215,21 @@ def main_crawler(keywords: list[str], start_datetime: str, end_datetime: str, da
-        urls.update(retried_urls)
-    logger.info(f"Extracted {len(urls)} URLs.")
-    total, success = load_each_post_with_keyword(keywords, urls, data_for_load_s3)
-    logger.info(f"Extracted {total} URLs out of {success} URLs.")
-    return total, success
+    total_count = 0
+    success_count = 0
+    for keyword in keywords:
+        urls, failed_page_list = get_list_of_post_url(start_datetime, end_datetime, list(range(0,50)), keyword)
+        logger.info(f"Found {len(urls)} URLs. Failed page URLs: {failed_page_list}")
+        if failed_page_list:
+            logger.info(f"Try failed page URLs.")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)